static/js/components/HamburgerAndLogo.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from "react"
+
+type Props = {
+  closeNavigation: Function
+}
+
+const HamburgerAndLogo = ({ closeNavigation }: Props) => (
```

Comment:
The prop should just be called `onMenuClick`/`onHamburgerClick`/etc. Clicking that link does not always close something